compose.yaml

```diff
@@ -0,0 +1,39 @@
+version: "3.4"
+
+x-common-variables: &common-variables
+  POSTGRES_USER: foo
+  POSTGRES_PASSWORD: bar
+  POSTGRES_DB: baz
+
+services:
```

Comment:
We should probably add the hub importer at some point, but could do it later if we're just trying to get the core demo running